static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx

```diff
@@ -0,0 +1,117 @@
+import {ModulePageProviders} from 'sentry/views/performance/database/modulePageProviders';
+import {SpanDescription} from 'sentry/views/starfish/components/spanDescription';
+import {DurationCell} from 'sentry/views/starfish/components/tableCells/durationCell';
+import {ThroughputCell} from 'sentry/views/starfish/components/tableCells/throughputCell';
+import {useSpanMetrics} from 'sentry/views/starfish/queries/useSpanMetrics';
+import {SpanFunction, SpanMetricsField} from 'sentry/views/starfish/types';
+import {DataTitles, getThroughputTitle} from 'sentry/views/starfish/views/spans/types';
+import {Block, BlockContainer} from 'sentry/views/starfish/views/spanSummaryPage/block';
+
+function ResourceSummary() {
+  const organization = useOrganization();
+  const {groupId} = useParams();
+  const {data: spanMetrics} = useSpanMetrics(groupId, {}, [
+    'avg(span.self_time)',
+    'spm()',
+    'span.op',
+    'span.description',
+  ]);
+
+  return (
+    <ModulePageProviders
+      title={[t('Performance'), t('Resources'), t('Resource Summary')].join(' — ')}
+    >
+      <Layout.Header>
+        <Layout.HeaderContent>
+          <Breadcrumbs
+            crumbs={[
+              {
+                label: 'Performance',
+                to: normalizeUrl(`/organizations/${organization.slug}/performance/`),
+                preservePageFilters: true,
+              },
+              {
+                label: 'Resources',
+                to: normalizeUrl(
+                  `/organizations/${organization.slug}/performance/browser/resources/`
+                ),
+                preservePageFilters: true,
+              },
+              {
+                label: 'Resource Summary',
+              },
+            ]}
+          />
+
+          <Layout.Title>
+            {t('Resources')}
+            <FeatureBadge type="alpha" />
+          </Layout.Title>
+        </Layout.HeaderContent>
+      </Layout.Header>
+
+      <Layout.Body>
+        <Layout.Main fullWidth>
+          <HeaderContainer>
+            <PaddedContainer>
+              <PageFilterBar condensed>
+                <ProjectPageFilter />
+                <DatePageFilter alignDropdown="left" />
+              </PageFilterBar>
+            </PaddedContainer>
+            <BlockContainer>
+              <Block title={DataTitles.avg}>
+                <DurationCell
+                  milliseconds={spanMetrics?.[`avg(${SpanMetricsField.SPAN_SELF_TIME})`]}
+                />
+              </Block>
+              <Block title={getThroughputTitle('http')}>
+                <ThroughputCell
+                  rate={spanMetrics?.[`${SpanFunction.SPM}()`] * 60}
+                  unit={RateUnits.PER_SECOND}
+                />
+              </Block>
+            </BlockContainer>
+          </HeaderContainer>
+          <DescriptionContainer>
+            <SpanDescription span={spanMetrics} />
```

Comment:
l: should be bump up the font size of the description and make it bold? An `h3` might work well.

Reply:
Good call, I changed it to an h3!